### ISSUE TITLE
Deprecation warnings after upgrading to react@16.6+ in StrictMode

### ISSUE THREAD

**Issue by franklixuefei** [DevExpress/devextreme-reactive#1602]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Current Behaviour
I saw the following deprecation warnings after upgrading to react@16.6+ with StrictMode:
1.
```
Warning: Legacy context API has been detected within a strict-mode tree: 
    in StrictMode
```
2.
```
index.js:1452 Warning: Legacy context API has been detected within a strict-mode tree: 
    in StrictMode

Please update the following components: Action, DragDropProvider, DragSource, DropTarget, Getter, InputBase, Plugin, PluginHost$$1, PluginIndexer, PluginIndexerContext, Select, Table, TableBody, TableCell, TableHead, TableRow, Template, TemplateConnector, TemplatePlaceholder
```

### Environment

- devextreme-reactive: xxx
- react: 16.6
- browser: xxx
- bootstrap: none
- react-bootstrap: none
- material-ui: 3.4.0

**Comment by SergeyAlexeev** [comment]
Hi,
 
We are already aware of this problem. The fix will be available with the next release.

**Comment by ushkal** [comment]
Hi, 

We've fixed legacy context API warnings in version 1.9.1. We will get rid of the "findDOMNode is deprecated" warning in the next major version since it requires introducing a breaking change.
